src/load.jl

```diff
@@ -0,0 +1,217 @@
+using Ghost
+using Ghost: Tape, Input, mkcall, Variable, V
+
+
+struct ONNXCtx
+    name2var::Dict{String, Variable}
+    backends::Vector{Symbol}
+    exec::Bool
+end
+
+ONNXCtx(backends; exec=true) = ONNXCtx(Dict(), backends, exec)
+ONNXCtx(;exec=true) = ONNXCtx(Dict(), [:ONNX], exec)
+
+# TODO: implement rebind_context!()
+
+"""
+    getindex(tape::Tape{ONNXCtx}, onnx_name::String)
+
+Get operation on the tape using the name in ONNX graph
+"""
+Base.getindex(tape::Tape{ONNXCtx}, onnx_name::String) =
+    tape[tape.c.name2var[onnx_name]]
+
+###############################################################################
+#                               Operations                                    #
+###############################################################################
+
+
+mrev(x) = x
+mrev(x::AbstractVector) = reverse(x)
+prev(x) = x
+prev(x::AbstractVector) = reshape(permutedims(reverse(reshape(x, length(x) ÷ 2,:);dims=1)),:)
+
+
+# mrev = maybe reverse. prev = rearrange padding, e.g. (1,2,1,2) => (2,2,1,1) or (1,2,3,1,2,3) => (3,3,2,2,1,1)
```

Comment:
Move this comment above?

Reply:
This is a direct copy of [code in ONNXNaiveNASflux](https://github.com/DrChainsaw/ONNXNaiveNASflux.jl/blob/master/src/deserialize/ops.jl#L65-L75), in another branch I've moved all such stuff to `conversions.jl` and plan to use unified API `onnx2julia()` / `julia2onnx()` to handle transformations of tensors, attribute names and attribute values. If you on't mind, let's move this discussion to that future PR.